src/main/java/com/fundynamic/d2tm/game/map/ShroudFacingDeterminer.java

```diff
@@ -0,0 +1,61 @@
+package com.fundynamic.d2tm.game.map;
+
+import com.fundynamic.d2tm.graphics.ShroudFacing;
+
+public class ShroudFacingDeterminer {
+
+    private static final int BIT_MASK_NONE = 0;
+    private static final int BIT_MASK_TOP = 0x0008;
+    private static final int BIT_MASK_RIGHT = 0x0004;
+    private static final int BIT_MASK_BOTTOM = 0x0002;
+    private static final int BIT_MASK_LEFT = 0x0001;
+
+    private static final ShroudFacing[] FACINGS = {
```

Comment:
its not that I don't agree, but merely that Dune 2 never had these other facings (i think).